bibliograph/parsing/tests/test_endnote_parser.py

```diff
@@ -48,6 +64,22 @@ def test_import(self):
             self.failIf(hasattr(entries.entries[0],'keywords')
                         and entries.entries[0].keywords,
                         'Parsed non existing keywords')
+
+        else:
+            print """\nOne or more transformationtool was not found!
+please make sure bibutils is installed to run all tests. """
+            print ("-" * 20) + "\n"
+
+    @expected_failure
```

Comment:
strangely enough, when using the apt package for bibutils, which is v4.12, this now passes.

So it seems like there is a bug that occurs on in bibutils sometime after v4.12 .  All of the releases after v4.12 are now in https://github.com/jayvdb/bibutils-archive , so we should be able to find the problem